src/app/helptext/task-calendar/replication/replication-wizard.ts

```diff
@@ -4,54 +4,80 @@ export default {
     step1_label: T('What and Where'),
 
     exist_replication_placeholder: T('Load Previous Replication Task'),
-    exist_replication_tooltip: T(''),
+    exist_replication_tooltip: T('Use settings from a saved replication.'),
 
     source_datasets_from_placeholder: T('Source Datasets'),
-    source_datasets_from_tooltip: T(''),
+    source_datasets_from_tooltip: T('Location of datasets with snapshots \
```

Comment:
This is not really a location, but the datasets themselves.
```suggestion
    source_datasets_from_tooltip: T('Datasets with snapshots \
```

Reply:
It is a location for this field - the only options in the drop-downs are the locations of the datasets